cmake - ONLY_CBLAS=1 fails utest

cblas only via cmake fails on utest

Install settings
`cmake -DCMAKE_INSTALL_PREFIX=/usr -DFC=/usr/bin/gcc10-gfortran -DCMAKE_BUILD_TYPE=Release -DONLY_CBLAS=1 ..
  cmake --build .
  cmake --build . --target install`

Utest Log
`==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_min_smin_negative_run: error: undefined reference to 'smin_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_min_smin_zero_run: error: undefined reference to 'smin_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_min_dmin_positive_run: error: undefined reference to 'dmin_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_max_smax_negative_run: error: undefined reference to 'smax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_max_smax_zero_run: error: undefined reference to 'smax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_min.c.o:test_min.c:function __ctest_max_dmax_positive_run: error: undefined reference to 'dmax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_amax.c.o:test_amax.c:function __ctest_amax_samax_run: error: undefined reference to 'samax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_amax.c.o:test_amax.c:function __ctest_amax_damax_run: error: undefined reference to 'damax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_ismin.c.o:test_ismin.c:function __ctest_ismin_positive_step_2_run: error: undefined reference to 'ismin_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_ismin.c.o:test_ismin.c:function __ctest_ismin_negative_step_2_run: error: undefined reference to 'ismin_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_ismin.c.o:test_ismin.c:function __ctest_ismax_positive_step_2_run: error: undefined reference to 'ismax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_ismin.c.o:test_ismin.c:function __ctest_ismax_negative_step_2_run: error: undefined reference to 'ismax_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rotmg.c.o:test_rotmg.c:function __ctest_drotmg_rotmg_run: error: undefined reference to 'drotmg_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rotmg.c.o:test_rotmg.c:function __ctest_drotmg_rotmg_issue1452_run: error: undefined reference to 'drotmg_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rotmg.c.o:test_rotmg.c:function __ctest_drotmg_rotmg_D1eqD2_X1eqX2_run: error: undefined reference to 'drotmg_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rotmg.c.o:test_rotmg.c:function __ctest_drotmg_drotmg_D1_big_D2_big_flag_zero_run: error: undefined reference to 'drotmg_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rot.c.o:test_rot.c:function __ctest_rot_drot_inc_0_run: error: undefined reference to 'drot_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rot.c.o:test_rot.c:function __ctest_rot_zdrot_inc_0_run: error: undefined reference to 'zdrot_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rot.c.o:test_rot.c:function __ctest_rot_srot_inc_0_run: error: undefined reference to 'srot_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_rot.c.o:test_rot.c:function __ctest_rot_csrot_inc_0_run: error: undefined reference to 'csrot_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_axpy.c.o:test_axpy.c:function __ctest_axpy_daxpy_inc_0_run: error: undefined reference to 'daxpy_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_axpy.c.o:test_axpy.c:function __ctest_axpy_zaxpy_inc_0_run: error: undefined reference to 'zaxpy_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_axpy.c.o:test_axpy.c:function __ctest_axpy_saxpy_inc_0_run: error: undefined reference to 'saxpy_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_axpy.c.o:test_axpy.c:function __ctest_axpy_caxpy_inc_0_run: error: undefined reference to 'caxpy_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_dsdot.c.o:test_dsdot.c:function __ctest_dsdot_dsdot_n_1_run: error: undefined reference to 'dsdot_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_dnrm2.c.o:test_dnrm2.c:function __ctest_dnrm2_dnrm2_inf_run: error: undefined reference to 'dnrm2_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_dnrm2.c.o:test_dnrm2.c:function __ctest_dnrm2_dnrm2_tiny_run: error: undefined reference to 'dnrm2_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_swap.c.o:test_swap.c:function __ctest_swap_dswap_inc_0_run: error: undefined reference to 'dswap_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_swap.c.o:test_swap.c:function __ctest_swap_zswap_inc_0_run: error: undefined reference to 'zswap_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_swap.c.o:test_swap.c:function __ctest_swap_sswap_inc_0_run: error: undefined reference to 'sswap_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_swap.c.o:test_swap.c:function __ctest_swap_cswap_inc_0_run: error: undefined reference to 'cswap_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_dotu.c.o:test_dotu.c:function __ctest_zdotu_zdotu_n_1_run: error: undefined reference to 'zdotu_'
==> amazon-ebs.base-ami-build: CMakeFiles/openblas_utest.dir/test_dotu.c.o:test_dotu.c:function __ctest_zdotu_zdotu_offset_1_run: error: undefined reference to 'zdotu_'
==> amazon-ebs.base-ami-build: collect2: error: ld returned 1 exit status
==> amazon-ebs.base-ami-build: gmake[2]: *** [utest/openblas_utest] Error 1
==> amazon-ebs.base-ami-build: gmake[1]: *** [utest/CMakeFiles/openblas_utest.dir/all] Error 2
==> amazon-ebs.base-ami-build: gmake: *** [all] Error 2`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -321,7 +321,7 @@ if (NOT NOFORTRAN)
  if (NOT ONLY_CBLAS)
   # Build test and ctest
   add_subdirectory(test)
- endif 
+ endif() 
   if (BUILD_TESTING)
     add_subdirectory(lapack-netlib/TESTING)
   endif()

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -311,19 +311,25 @@ endif()
 
 #if (MSVC OR NOT NOFORTRAN)
 if (NOT NO_CBLAS)
+  if (NOT ONLY_CBLAS)
   # Broken without fortran on unix
-  add_subdirectory(utest)
+    add_subdirectory(utest)
+endif()
 endif()
 
 if (NOT NOFORTRAN)
+ if (NOT ONLY_CBLAS)
   # Build test and ctest
   add_subdirectory(test)
+ endif 
   if (BUILD_TESTING)
     add_subdirectory(lapack-netlib/TESTING)
   endif()
 endif()
   if(NOT NO_CBLAS)
+   if (NOT ONLY_CBLAS)
     add_subdirectory(ctest)
+   endif()
   endif()
   if (CPP_THREAD_SAFETY_TEST OR CPP_THREAD_SAFETY_GEMV)
     add_subdirectory(cpp_thread_test)

